### PR TITLE
Bank file reading

### PR DESCRIPTION
**Pull request by dissco6**

### CODE REVIEW

File: back-end-side/Controllers/ReportReader.cs [hunk 1]
```diff
@@ -1,17 +1,92 @@
...
+                        (records.ElementAt(i).Seller.Equals("") && records.ElementAt(i).Purpose.Equals("Apyvarta")) ||
+                        records.ElementAt(i).PaymentType.Equals("K"))
+                    {
+                        records.RemoveAt(i);
+                    }
+                }
+
+                return records;
+            }
+            else if (bank == 1)
```
Comment (shilingas) [inline back-end-side/Controllers/ReportReader.cs:43]: here you can also change to if (bank)

File: back-end-side/Controllers/ReportReader.cs [hunk 2]
```diff
@@ -1,17 +1,92 @@
...
+                        records.RemoveAt(i);
+                    }
+
+                    records.ElementAt(i).Amount /= 100;
+                }
+
+                return records;
+            } else
+            {
+                Console.WriteLine("Invalid bank ID");
```
Comment (shilingas) [inline back-end-side/Controllers/ReportReader.cs:85]: what we will do if the bank is not swed/paysera/seb?

File: back-end-side/Controllers/ReportReader.cs [hunk 3]
```diff
@@ -1,17 +1,92 @@
...
-            return records;
+            // swedbank = 0, paysera = 1, seb = 2
+            int bank = 0;
+
+            var DelimiterToSemicolon = new CsvConfiguration(CultureInfo.InvariantCulture)
+            {
+                Delimiter = ";",
+            };
+
+            if (bank == 0)
```
[the remark this answers is not in the document]
Comment (dissco6) [reply back-end-side/Controllers/ReportReader.cs:21]: Variable bank values could be not only 0 or 1, so I believe, that code is more clear, when variable is compared with the exact value

File: back-end-side/Controllers/ReportReader.cs (hunk 1 above)
Comment (dissco6) [reply back-end-side/Controllers/ReportReader.cs:43]: Same here

**Pull request merged**